src/app_uart.c

```diff
@@ -248,6 +249,7 @@ void app_uart_parser (void * p_data, uint16_t data_len)
 
     if (RD_SUCCESS == err_code)
     {
+      printf("company_filter: %d\r\r", uart_payload.params.all_params.bools.fltr_tags);
```

Comment:
Use ri_log instead. That allows for setting log levels and selecting if logs should go to RTT or UART